lecture/views.py

```
@@ -3,21 +3,57 @@
-from .forms import LectureForm, LecturePDFForm
-from .models import Lecture
+from .forms import LectureForm, LecturePDFForm,LectureVideoForm
+from .models import Lecture,LectureVideo
 from courses.models import Courses
 from django.views.decorators.http import require_http_methods
+from django.http import FileResponse
+from django.core.exceptions import ValidationError
+from django.http import HttpResponseBadRequest
 
-
+def validate_file(file, type):
```

Ruff (0.8.2)

Missing return type annotation for public function `validate_file`

Add return type annotation: `None`

(ANN201)